src/tools/align/components/AlignForm.tsx

```diff
@@ -54,6 +56,7 @@ import {
 
 import sticky from '../../../shared/styles/sticky.module.scss';
 import '../../styles/ToolsForm.scss';
+import { logger } from '@sentry/utils';
```

Comment:
I think this wants to be something like `import * as logging from './shared/utils/logging';`